src/app/vertextool/qgsvertexeditor.cpp

```diff
@@ -457,6 +459,20 @@ void QgsVertexEditor::keyPressEvent( QKeyEvent *e )
     // Override default shortcut management in MapCanvas
     e->ignore();
   }
+  else if ( e->matches( QKeySequence::Copy ) )
+  {
+    QString text;
+    QItemSelectionRange range = mTableView->selectionModel()->selection().first();
```

Comment:
better check that the selection is not empty first, or calling .first() will crash